config_management.rst

```diff
@@ -387,11 +389,10 @@ with the content below:
     - name: /etc/apache2/sites-enabled/www.opsschool.org
     - source: salt://vhosts/conf/www.opsschool.org
 
-Above, we include already described state of the Apache service and extend it
-to include our configuration file. Notice we use a new directive ``watch``
-to describe our state as being dependent on what changes the configuration
-file triggers. This way, if a newer version of the same file is deployed, it
-should restart the Apache service.
+Above, we include the existing Apache service state and extend it to include
+our configuration file. The ``watch`` requisite indicates that the Apache
```

Comment:
What's a requisite in salt? This is the first time the term comes up in this section.